packages/gestalt/src/Icon.css

```diff
@@ -7,3 +7,8 @@
 .iconBlock {
   composes: block from "./Layout.css";
 }
+
+html[dir="rtl"] .rtlSupport {
+  /* Flip the icon for right-to-left language locales */
+  transform: rotateY(180deg);
+}
```

Comment:
@selena-zeng could we add an extra breaking line at the end of this file?

![image](https://user-images.githubusercontent.com/127199/72906537-3dca4700-3ce7-11ea-8e5c-b29dc44c850e.png)